server/src/main/java/org/apache/iotdb/db/query/udf/datastructure/tv/ElasticSerializableTVList.java

```diff
@@ -100,6 +110,10 @@ public int size() {
     return size;
   }
 
+  public boolean isNull(int index) throws IOException {
+    return nullIndices.contains(index);
```

Comment:
I'm more concerned about the boxing and unboxing time overhead of the `index` than the space footprint of the `nullIndices`. 

Maybe it would be better to add a `Bitmap` field to `SerializableTVList`?

Reply:
I have optimized the implementation of `ElasticSerializableTVList` and updated the test cases. Please take a look, especially the changes of the logics of memery calculation, memory control and cache eviction.

Reply:
`ElasticSerializableTVList` and `ElasticSerializableRowRecordList` are all updated.